### DISCUSSION TITLE
Bug Report: Azimuth mode, first stamp does not follow pen direction

### DISCUSSION THREAD

**Discussion by miya-sans** [keiwando/toonsquid#315] [Q&A]
Device: iPad Pro (12.9 inch) (6th generation)
iOS Version: 16.4.1
Version: 1.9

https://github.com/keiwando/toonsquid/assets/42623210/348c0c89-4055-4c9b-a02c-cc681d6c921f

Using Rotation Behavior: -100%, which activates Azimuth mode, the first stamp doesn't follow the pen direction, but a non-rotated stamp. In the demonstration video, I tried making strokes in various directions while rotating the pen. This issue persists when rotating/flipping the shape and reducing the spacing. Setting the Rotational Behavior to a value closer to 0% lessens the effect. 

Having made the brush in version, 1.8.6, I don't believe this bug existed in that version, but this is anecdotal. Having Azimuth mode as a toggle in the shape settings, like Procreate will be more streamline; as setting a negative rotation value doesn't intuitively produce an Azimuth result, this feature would seemingly be unknown, unless consulting the documentation. 

The follow effect from Rotational Behavior being a positive value doesn't seem to be self-descriptive either, Procreate labels when this effect is used when dragging the slider past 100%, this setting is also integrated in the Base Rotation slider. See attached image. In fact design-wise, I feel like the settings in the Rotation tap could be moved to the Shape tab, seeing how Flip X/Y are already there. 

In addition, I've attached the file for the brush below:

[flat_brush.zip](https://github.com/keiwando/toonsquid/files/11584833/flat_brush.zip)

![Screenshot 2023-05-28 at 23 34 21](https://github.com/keiwando/toonsquid/assets/42623210/1e0f0a12-9344-493f-a18c-fc3a284735c9)

**Comment by keiwando** [comment]
This should now be fixed with the 1.9.4 update. Thanks for reporting it!
